pkg/telemetry/telemetryserviceinternal.go

```diff
@@ -103,6 +109,30 @@ func (t *telemetryServiceInternal) SendAnalytics() {
 	}
 }
 
+func (t *telemetryServiceInternal) CleanupWorkers() {
+	t.workersMu.RLock()
+	workers := t.workers
+	t.workersMu.RUnlock()
+
+	for _, worker := range workers {
+		if worker == nil {
+			continue
+		}
+
+		closedAt := worker.ClosedAt()
+		if !closedAt.IsZero() && time.Since(closedAt) > workerCleanupWait {
+			pID := worker.ParticipantID()
+			logger.Infow("reaping analytics worker for participant", "pID", pID)
```

Comment:
should this be a debug log?

Reply:
Will change it. Put it at `Infow` just in case we start seeing issues. But, change is safe enough. Will make it `Debugw`.